person.rb

```diff
@@ -15,7 +21,7 @@ def can_use_services?
 
   private
 
-  def of_age?
-    @age >= 18
+  def correct_name
+    @name
   end
 end
```

Comment:
- You have done great so far but there are few issues you need to fix before moving to the next milestone. :smile: :wave: 
- [ ] The method `correct_name` is not a private method and therefore it should be moved above the `private` keyword. :+1: